packages/babel-plugin-transform-es2015-template-literals/src/index.js

```diff
@@ -18,7 +18,9 @@ export default function ({ types: t }) {
         let raw = [];
 
         for (const elem of (quasi.quasis: Array)) {
-          strings.push(t.stringLiteral(elem.value.cooked));
+          if (elem.value.cooked !== null) {
```

Comment:
Cool, wanted to verify my understanding then. So @hzoo this should do

    const value = elem.value.cooked == null
      ? t.unaryExpression("void", t.numericLiteral(0))
      : t.stringLiteral(elem.value.cooked);
    strings.push(value);